its/ruling/src/test/resources/expected_extended/salt/python-S930.json

```diff
@@ -29,7 +32,22 @@
 6382,
 6383,
 ],
+'salt:salt/modules/win_service.py':[
+216,
+218,
+],
+'salt:salt/modules/win_useradd.py':[
+334,
+705,
+],
+'salt:salt/spm/__init__.py':[
+1114,
+1117,
```

Comment:
These are within an `except` block and are valid in Python 2.
The correct Python 3 call is in the `try` block so this seems to be an intended fallback (FP).
We should probably exclude `except` blocks from this rule.